documentation/en/user/source/configuration/storage.rst

```diff
@@ -272,6 +271,173 @@ GeoServer ``topp:states`` sample layer on a fictitious ``my-geowebcache-bucket``
       })
     });
 
+MBTiles Blob Store
+++++++++++++++++++
+
+This blob store allow us to store tiles using the `MBTiles <https://github.com/mapbox/mbtiles-spec/blob/master/1.1/spec.md>`_ specification (version 1.1) which defines a schema for storing tiles in an `SQLite <https://www.sqlite.org/>`_ database with some restrictions regarding tiles formats and projections.
+
+MBTiles specification only supports JPEG and PNG formats and projection EPSG:3857 is assumed. The implemented blob store will read and write MBTiles files compliant with the specification but will also be able to write and read MBTiles files that use others formats and projections.
+
+Using the MBTiles blob store will bring several benefits at the cost of some performance loss. We will be able to store our data with a significantly smaller number of files, which will help us avoid file system headaches and help us managing our data. In some cases the stored data will be more compacted reducing the size of the data on disk.
```

Comment:
The documentation tries not to use "we" or "you" but use a generic third person, this section should be reworked.
E.g. "The MBTiles storage uses a significantly smaller number of files, which results in easier data handling (e.g., backups, moving tiles between environments)".